token/token.go

```diff
@@ -118,6 +118,11 @@ const (
 	LAMBDA // => lambda short cut: `a,b => a+b` alias for `func(a,b) {a+b}`
 	DEFINE // := (force create new variable instead of possible ref to upper scope)
 
+	SUMASSIGN
+	SUBASSIGN
+	PRODASSIGN
+	DIVASSIGN
```

Comment:
needs at least a comment to say the order has to be same as PLUS MINUS ASTERISK SLASH